e2e-tests/steps/cancelInProgressApplication.ts

```diff
@@ -0,0 +1,12 @@
+import { Page } from '@playwright/test'
+import { ApplyPage } from 'e2e-tests/pages/apply'
+
+export const cancelAnApplication = async (page: Page, name: string) => {
+  const cancelPage = await ApplyPage.initialize(page, `Are you sure you would like to cancel ${name}'s application?`)
```

Comment:
Are we reusing the Page object here just to help with testing? I wouldn't have thought to do that, very nice!